tests/functional/conftest.py

```diff
@@ -25,3 +27,53 @@ def playwright_install(request):
         command.extend(["--with-deps"])
     with capmanager.global_and_fixture_disabled():
         subprocess.run(command, check=True)
+
+
+TEST_USERS = {
+    "test_output_checker": {
+        "id": "test_output_checker",
+        "username": "test_output_checker",
+        "workspaces": [],
+        "output_checker": True,
+    },
+    "test_researcher": {
+        "id": "test_researcher",
+        "username": "test_researcher",
+        "workspaces": ["test-dir1"],
+        "output_checker": False,
+    },
+}
```

Comment:
Minor stylistic thing, but given that you've got separate `output_checker_user` and `researcher_user` fixtures which wrap `login_as_user` you could lose this global config entirely and have each of those fixtures pass the relevant config e.g:
```py
@pytest.fixture
def output_checker_user(login_as_user):
    yield login_as_user(
        {
            "id": "test_output_checker",
            "username": "test_output_checker",
            "workspaces": [],
            "output_checker": True,
        }
    )
```

It feels like a unnecessary layer of indirection otherwise.

Reply:
Yeah, I could see either approach working in different contexts. Feel free to do it however you think makes most sense.